src/constants/tokens.js

```diff
@@ -180,7 +180,7 @@ const UBI_TOKEN = {
   typeId: 6,
   icon: 'https://raw.githubusercontent.com/rsksmart/rsk-contract-metadata/master/images/ubi.png',
   30: { symbol: 'rUBI', address: '0x70566d8541beabe984c8babf8a816ed908514ba8', decimals: 18 },
-  31: { symbol: 'rUBI', address: '0x1c85316116131133cf6fdb0cde836b23035ea775', decimals: 18 },
```

Comment:
so we had rUBI duplicated? I guess the frontend table shows unique symbol values since I don't see it twice in the frontend

Reply:
They are on diferent networks so it's not duplicated, the error is the displaying Symbol for ir on rskTestnet, it should have been rKovUBI instead of rUBI